### ISSUE TITLE
Filter subscriptions to the existing tags or notes

### ISSUE THREAD

**Issue by gautamig54** [publiclab/plots2#5555]
Currently, I am able to subscribe the tags that don't even exist by typing them to the enter tags input section.

![Screen Shot 2019-04-20 at 5 05 48 PM](https://user-images.githubusercontent.com/35326753/56456772-a2d6c200-638e-11e9-8c81-578515d820b2.png)

I think we should filter out such tags and display an alert message saying that the tag/note does not exists and thus cannot be subscribed to.

**Comment by gautamig54** [comment]
@publiclab/plots2-reviewers What do you think about this?

**Comment by IshaGupta18** [comment]
So you are able to just add the tag or subscribe to it? If you are able to
subscribe to it, then this is an actual problem. We can work on this
together if you like.

**Comment by gautamig54** [comment]
@IshaGupta18  I think we are able to subscribe to it.

**Comment by IshaGupta18** [comment]
Yes we are! Okay we can brainstorm on it together, on how to go about it.
What do you think?

**Comment by gauravano** [comment]
Okay, one case is where irrespective of the presence of tag, user wants to get subscribed (Just a random thought). @jywarren this case should be considered? If no, then, we are good to go here. Thanks!

**Comment by jywarren** [comment]
Hmm, how does it currently happen -- does it create the tag and subscribe
to it? Since anyone can make tags, what happens if you subscribe to
'unique-tag' and then tag a post with the same tag?

**Comment by gauravano** [comment]
> does it create the tag and subscribe to it?

Yes!

> Since anyone can make tags, what happens if you subscribe to 'unique-tag' and then tag a post with the same tag?

It works fine as we check for the existence of tag while adding.

Also, while experimenting with tags, I found that when I try to subscribe to same tag again it shows "you are now following tagname" but in subscription_controller we handle such cases so we need to fix that too.

Thanks!

**Comment by gautamig54** [comment]
Yes @gauravano. Regarding the already subscribed tag, I've created another issue #5556. 
If a user is already subscribed, it will show already subscribed instead of the present message.

**Comment by gauravano** [comment]
Yes, it's a bug, I have edited your issue a bit just now. Thanks for pointing!
